config/splat.us.main.yaml

```diff
@@ -35,7 +35,7 @@ segments:
       - [0x8F8, .rodata, psxsdk/libgpu/prim]
       - [0x9C8, .rodata, psxsdk/libgpu/sys]
       - [0xBC4, rodata, psxsdk/libapi/a22] # aVsyncTimeout
-      - [0xC08, .rodata, psxsdk/libapi/l10]
+      - [0xC08, .rodata, psxsdk/libetc/intr]
```

Comment:
I believe these splits are correct. BTW I imported the current libetc work into the other repo, there's various fixes like importing all the data and fixing some static / non-static differences. https://github.com/sozud/psy-q-decomp/tree/master/src/etc